PermissionsEx/permissions.yml

```diff
@@ -179,7 +184,8 @@ groups:
     options:
       rank: '10'
   members:
-    permissions: []
+    permissions: 
+    - plots.plot.1
```

Comment:
This is not the correct member group (`members` vs `member`). This group should be removed later at a cleanup.